Source/DafnyCore/Rewriters/RefinementTransformer.cs

```diff
@@ -1058,8 +1058,7 @@ List<Statement> MergeStmtList(List<Statement> skeleton, List<Statement> oldStmt,
                 // that the condition is inherited.
                 var e = refinementCloner.CloneExpr(oldAssume.Expr);
                 var attrs = refinementCloner.MergeAttributes(oldAssume.Attributes, skel.Attributes);
-                body.Add(new AssertStmt(new BoogieGenerator.ForceCheckOrigin(skel.Origin),
-                  e, skel.Label, new Attributes("_prependAssertToken", new List<Expression>(), attrs)));
```

Comment:
I like getting rid of hacky attributes is nice, but have you considered all implications of that? If so, just resolve this conversation, don't bother replying to it.

Reply:
This attribute was only used in one location and its use was not too complicated.